bot/constants.py

```diff
@@ -13,5 +16,28 @@
 LOG_FILE = pathlib.Path("log/gurkbot.log")
 
 
+class Colours:
+    blue = 0x0279FD
+    bright_green = 0x01D277
+    dark_green = 0x1F8B4C
+    orange = 0xE67E22
+    pink = 0xCF84E0
+    purple = 0xB734EB
+    soft_green = 0x68C290
+    soft_orange = 0xF9CB54
+    soft_red = 0xCD6D6D
+    yellow = 0xF9F586
+
+
 class Channels(NamedTuple):
-    devlog = int(os.environ.get("CHANNEL_DEVLOG", 789431367167377448))
+    announcements = int(os.getenv("CHANNEL_ANNOUNCEMENTS", 123456789123456789))
+    devalerts = int(os.getenv("CHANNEL_DEVALERTS", 123456789123456789))
```

Comment:
```suggestion
    devalerts = int(os.getenv("CHANNEL_DEVALERTS", 796695123177766982))
```
I've created a `#dev-alerts` channel in the main server now.

Reply:
Please commit this.